frontend/src/services/UserService.ts

```diff
@@ -0,0 +1,29 @@
+import http from '../util/http';
+import User from '../types/User';
+
+export default class UserService {
+  static async getUsers(): Promise<User[]> {
+    const response = await http.get('/users');
+    return response.data;
+  }
+
+  static async getUser(id: number): Promise<User> {
+    const response = await http.get(`/users/${id}`);
+    return response.data;
+  }
+
+  static async createUser(user: User): Promise<User> {
```

Comment:
Does the promise type have to be user for all endpoints?

Reply:
ggod point